packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -60,6 +62,21 @@ enum CallEndReasons {
 }
 
 const getCallEndReason = (call: CallState): CallEndReasons => {
+  const remoteParticipantsEndedArray = Array.from(Object.values(call.remoteParticipantsEnded));
+  /**
+   * Handle the special case in a PSTN call where removing the last user kicks the caller out of the call.
```

Comment:
So PSTN call can't hold the call for just one participant? Every PSTN call needs at least 2 people?

Reply:
Correct